examples/ui/ui_material.rs

```diff
@@ -33,18 +35,26 @@ fn setup(
         .with_children(|parent| {
             let banner_scale_factor = 0.5;
             parent.spawn((
+                Node {
+                    position_type: PositionType::Absolute,
+                    width: Val::Px(905.0 * banner_scale_factor),
+                    height: Val::Px(363.0 * banner_scale_factor),
+                    border: UiRect::all(Val::Px(20.)),
+                    ..default()
+                },
                 MaterialNode(ui_materials.add(CustomUiMaterial {
                     color: LinearRgba::WHITE.to_f32_array().into(),
                     slider: 0.5,
                     color_texture: asset_server.load("branding/banner.png"),
                     border_color: LinearRgba::WHITE.to_f32_array().into(),
+                    corner_color: LinearRgba::NONE.to_f32_array().into(),
                 })),
-                Node {
-                    position_type: PositionType::Absolute,
-                    width: Val::Px(905.0 * banner_scale_factor),
-                    height: Val::Px(363.0 * banner_scale_factor),
-                    border: UiRect::all(Val::Px(10.)),
-                    ..default()
+                BorderRadius::all(Val::Px(20.)),
+                // UI material nodes can have outlines and shadows like any other UI node
+                Outline {
+                    width: Val::Px(2.),
+                    offset: Val::Px(100.),
```

Comment:
```suggestion
                    offset: Val::Px(10.),
```

The outline is really far from the node and for a while I was wondering what this extra blue border was 😅 You could change it to 10 and maybe it looks more like an outline:

<img width="516" alt="image" src="https://github.com/user-attachments/assets/a60415af-8418-45ca-8a67-be5ec661bc9e" />

Non-blocking.